server/tests/flight.test.js

```diff
@@ -0,0 +1,28 @@
+import request from 'supertest-as-promised';
```

Comment:
@jonathanbarton  @morrissinger  I have added the tests for this endpoint.
I was planning to use https://github.com/williamkapke/mongo-mock for mocking mongoDb for the tests.Let me know if you guys have another mock db in mind.